cpp/net/connection_pool.cc

```diff
@@ -279,14 +298,22 @@ void ConnectionPool::Put(unique_ptr<ConnectionPool::Connection>&& conn) {
 void ConnectionPool::Cleanup() {
   unique_lock<mutex> lock(lock_);
   cleanup_scheduled_ = false;
+  const system_clock::time_point cutoff(
+      system_clock::now() -
+      seconds(FLAGS_connection_pool_max_unused_age_seconds));
 
-  // std::map<HostPortPair, std::deque<unique_ptr<Connection>>> conns_;
+  // std::map<HostPortPair, std::deque<TimestampedConnection>> conns_;
```

Comment:
I think that was put there as a reference for what the `auto` in the for loop below would be.